portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,81 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+
+	var dropdown = document.getElementById("<%= namespace %>");
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	function onButtonKeyDown(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+			event.preventDefault();
+			list.classList.add('show');
+			button.setAttribute('aria-expanded', 'true');
+			var selectedOption = null;
+
+			Array.from(options).find(function(option) {
+				if (option.id == selectedItemValue) {
+					selectedOption = option;
+				}
+			});
+
+			if (selectedOption) {
+				selectedOption.focus();
+			}
+		}
+	}
+
+	button.addEventListener('keydown',onButtonKeyDown );
+
+	function handleKeyEvents(event, currIndex) {
+		if (event.key === 'ArrowDown') {
+			event.preventDefault();
+
+			if (currIndex < options.length - 1) {
+				options[currIndex + 1].focus();
+			}
+		} else if (event.key === 'ArrowUp') {
+			event.preventDefault();
+
+			if (currIndex > 0) {
+				options[currIndex - 1].focus();
+			}
+		}
+	}
+
+	options.forEach(function(option, index) {
+		option.addEventListener('keydown', function(event) {
+			handleKeyEvents(event, index);
+		});
```

Comment:
Just a warning about this, if you create an anonymous function here and then call `handleKeyEvents` it will not work when you try to remove the event, the registration and removal of the event works based on the reference in memory to know which method to remove.

```suggestion
		option.addEventListener('keydown', handleKeyEvents);
```

As you need the index, you can modify the implementation of `handleKeyEvents` to get the active item and check its index with the options array, for example.

```js
const currentIndex = options.indexOf(document.activeElement);
```

Reply:
I had to change this a little bit because optiosn is a _NodeList_ so a made it like this: 

`const currentIndex = options.indexOf(document.activeElement);`